pkg/manager/manager.go

```diff
@@ -923,6 +923,7 @@ func (m *jobManager) ResolveImageOrVersion(imageOrVersion, defaultImageOrVersion
 		imagestreams = append(imagestreams, namespaceAndStream{Namespace: "ocp-multi", Imagestream: "4.16-art-latest-multi", ArchSuffix: "-multi"})
 		imagestreams = append(imagestreams, namespaceAndStream{Namespace: "ocp-multi", Imagestream: "4.15-art-latest-multi", ArchSuffix: "-multi"})
 		imagestreams = append(imagestreams, namespaceAndStream{Namespace: "ocp-multi", Imagestream: "4.14-art-latest-multi", ArchSuffix: "-multi"})
+		imagestreams = append(imagestreams, namespaceAndStream{Namespace: "ocp-multi", Imagestream: "4.13-art-latest-multi", ArchSuffix: "-multi"})
```

Comment:
Since we currently know that the list of "Supported Hypershift Versions" (`HypershiftSupportedVersions`):
```
I0510 08:27:49.274026       1 manager.go:196] Hypershift Supported Versions: [4.13 4.14 4.15 4.16]
```
We can probably generate these values accordingly and avoid having to make these kinds of changes all together.

Reply:
That makes sense, I will make the change.